### PR TITLE
Use paramiko for ssh connection in test_wr_arp

### PR DESCRIPTION
**Pull request by wangxin**
### Description of PR

Summary:
Fixes # (issue)
The test_wr_arp.py test failed for two reasons:
1. After reboot, the ptf ssh key on dut host is lost. Executing remote command on dut from ptf failed.
2. This test script needs to do warm reboot on dut and would cause some error messages in syslog. The test will fail when loganalyzer is enabled.

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

#### What is the motivation for this PR?
The test_wr_arp script put ssh key of ptfhost on dut to have password less
ssh connection. Then the ptfhost can execute command remote on dut
through this ssh connection. The problem is that the ssh key saved in
~/.ssh/authorized_keys on dut host may not persist after warm reboot.
It would also be a problem when secure boot feature is introduced. This
PR updated the test_wr_arp script to use paramiko for ssh connection
and remote command execution between ptfhost and dut.

#### How did you do it?
* Update the ptf test script to use paramiko for SSH connection with dut
* Disabled loganalyzer because test_wr_arp need to do warm-reboot on dut

#### How did you verify/test it?
Test run the test_wr_arp.py script on latest 201911 image.

### CODE REVIEW

File: ansible/roles/test/files/ptftests/wr_arp.py [hunk 1]
```diff
@@ -60,19 +63,34 @@ def cmd(self, cmds):
...
+        stdout = stderr = []
+        return_code = 1
+
+        try:
+            client.connect(self.dut_ssh, username=self.dut_username, password=self.dut_password, allow_agent=False)
+            si, so, se = client.exec_command(cmd, timeout=30)
+            stdout = so.readlines()
+            stderr = se.readlines()
+            self.log('executed command {}, stdout={}, stderr={}'.format(cmd, stdout, stderr))
+            return_code = 0
```
Comment (bingwang-ms) [inline ansible/roles/test/files/ptftests/wr_arp.py:79]: Can we get return code of exec_command? Is it possible to ignore some error if we always return 0 here?

Comment (wangxin) [reply ansible/roles/test/files/ptftests/wr_arp.py:79]: The paramiko library uses pattern of sending cmd string and receiving returned results in streams. There is no return code returned by the exec_command method of the paramiko.SSHClient.

Comment (tahmed-dev) [reply ansible/roles/test/files/ptftests/wr_arp.py:79]: can we reuse this [class](https://github.com/Azure/sonic-mgmt/blob/master/ansible/roles/test/files/ptftests/device_connection.py)?

Comment (wangxin) [reply ansible/roles/test/files/ptftests/wr_arp.py]: Sure, pushed a new commit.

**Comment by wangxin** [comment]
retest this please

**Pull request merged**